### PR TITLE
add functionality for getting roadmaps and checkpoints

### PR DESCRIPTION
**Pull request by DarylMcCullough**

### CODE REVIEW

File: lib/roadmap.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            return @my_roadmap
+        end
+        headers = {
+            "Content-Type" => "application/json",
+            "authorization" => @auth_token
+        }
+        response = self.class.get("/roadmaps/#{id}", headers: headers)
+        body_text = response.body
+        @roadmap = JSON.parse(body_text)
+        @roadmap
```
Comment (brittanyjspringer) [inline lib/roadmap.rb]: This is redundant. The last value of the method is what is returned (in this case @roadmap).

**Review by brittanyjspringer — CHANGES_REQUESTED** [review]

**Comment by brittanyjspringer** [comment]
@DarylMcCullough so close, I just need one small change above. Really work on setting defaults!

File: lib/roadmap.rb (hunk 1 above)
Comment (DarylMcCullough) [reply lib/roadmap.rb]: I didn't know that an assignment statement returns the value that is being assigned.

Comment (brittanyjspringer) [reply lib/roadmap.rb]: It does - go, Ruby!

**Review by brittanyjspringer — APPROVED** [review]

**Pull request merged**